.circleci/config.yml

```diff
@@ -0,0 +1,188 @@
+version: 2
+jobs:
+    python-3.5: &build
+        docker:
+            - image: circleci/jdk8:0.1.1
```

Comment:
why this image?

Reply:
I wanted to use an image that was having trusty
This is practically the only circleci image that had trusty and had something that was required by us
Python images use stretch, jessie